### PR TITLE
If the location is NC, derive the zone from the operator

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: src/apply_manual_codes.py [hunk 1]
```diff
@@ -187,4 +187,15 @@ def apply_manual_codes(cls, user, data, coda_input_dir):
                         Metadata.get_call_location()).to_dict()
                 }, Metadata(user, Metadata.get_call_location(), time.time()))
 
+            # If the location is not coded, set the zone from the operator
```
Comment (lukechurch) [inline src/apply_manual_codes.py]: Would it be easy to factor this into another function? I'm interested in taking things which do logical imputation and moving them closer to coda and out of the core pipelines, to allow better interaction during the coding process

**Review by lukechurch — APPROVED** [review]

File: src/apply_manual_codes.py (hunk 1 above)
Comment (as2388) [reply src/apply_manual_codes.py]: It can be, but where would you like it to go (for now)?

Comment (lukechurch) [reply src/apply_manual_codes.py]: For now you can just put it as a private method in the same place. It'll help me in defining the API for the logical imputation and providing examples that can be cleanly migrated

Comment (as2388) [reply src/apply_manual_codes.py]: Moved to cls._impute_location_codes

**Pull request merged**